website/docs/r/dynamodb_table.html.markdown

```diff
@@ -83,10 +84,11 @@ The following arguments are supported:
 
 * `name` - (Required) The name of the table, this needs to be unique
   within a region.
+* `billing_mode` - (Optional) Controls how you are charged for read and write throughput and how you manage capacity. The valid values are `PROVISIONED` and `PAY_PER_REQUEST`. Defaults to `PROVISIONED`.
 * `hash_key` - (Required, Forces new resource) The attribute to use as the hash (partition) key. Must also be defined as an `attribute`, see below.
 * `range_key` - (Optional, Forces new resource) The attribute to use as the range (sort) key. Must also be defined as an `attribute`, see below.
-* `write_capacity` - (Required) The number of write units for this table
-* `read_capacity` - (Required) The number of read units for this table
+* `write_capacity` - (Optional) The number of write units for this table. If the `billing_mode` is `PROVISIONED`, this field is required.
```

Comment:
Does the API return a helpful error if read/write capacity is omitted with `PROVISIONED` billing mode?